### DISCUSSION TITLE
What fields are read from the .bib bibliogphy file entries?

### DISCUSSION THREAD

**Discussion by tcrnchs** [quarto-dev/quarto-cli#9151] [Q&A] [answered]
### Description

I have the following entry in a .bib file:

@book{Blankholm1991,
   author = {H. P. Blankholm},
   city = {Aarhus},
   publisher = {Aarhus University Press},
   title = {Intrasite spatial analysis in theory and practice},
   year = {1991},
}

The CSL file correctly processes "publisher", but the "city" field is not available which means the reference is displayed like this:

Blankholm, H. P. (1991). Intrasite spatial analysis in theory and practice. Aarhus University Press.

Rather than like this:

Blankholm, H. P. (1991). Intrasite spatial analysis in theory and practice. Aarhus : Aarhus University Press.

I can replace the "city" field in the CSL file with "publisher" and the reference is then displayed like this so it is not an issue with the CSL file:

Blankholm, H. P. (1991). Intrasite spatial analysis in theory and practice. Aarhus  University Press : Aarhus University Press.

Any ideas what I am missing here?

**Comment by mcanouil** [comment, marked as answer]
This is not related to Quarto, at least not directly.

This being said, CSL file defines what is read or not and how the elements are formatted.
You can find links to CSL and how to use them in the [documentation website](https://quarto.org/docs/authoring/footnotes-and-citations.html#sec-citations-style), then it is up to you to find the CSL file that suits your need.

**Reply by tcrnchs** [reply]
Thank you for your response. I will investigate the CSL documentation.

**Reply by cderv** [reply]
> This is not related to Quarto, at least not directly.

To be complete on this, This will be [Pandoc Citeproc](https://pandoc.org/MANUAL.html#citations) at play here.  You should try out how Pandoc itself behaves on this (without quarto).

**Reply by tcrnchs** [reply]
Thank you for this suggestion. I will investigate how Pandoc functions with this data.
